content/src/cloud/faq-ha-cloud/faq-nc-do-you-have-access-to-my-instance.md

```diff
@@ -6,4 +6,13 @@ zendesk:
   labels: cloud
 ---
 
-No, we don't. We will only exchange messages between your Home Assistant instance and the Google and Amazon voice assistants. The source code that handles incoming message from the cloud is in this Nabu Casa [GitHub repository](https://github.com/NabuCasa/hass-nabucasa/blob/master/hass_nabucasa/iot.py).
+No, we don't. We specifically designed our systems so that we have no access to your Home Assistant instance.
+
+- The connection is always securely established from your Home Assistant instance to our servers, never the reverse.
+- We only exchange encrypted messages between your instance and cloud services (like Google Assistant and Amazon Alexa).
+- We cannot access your Home Assistant instance, devices, or configuration.
+- We have no administrative tools or other means to reconnect a disconnected instance or to access your system.
```

Comment:
We do have a button to request remote access to be enabled from the account page?